scalikejdbc-core/src/main/scala/scalikejdbc/StatementExecutor.scala

```diff
@@ -37,8 +37,12 @@ case class StatementExecutor(
     extends LogSupport with UnixTimeInMillisConverterImplicits {
 
   import StatementExecutor._
+  import GlobalSettings.timeZoneConversion
 
   private[this] lazy val batchParamsList = new MutableList[Seq[Any]]
+  private[this] lazy val timeZoneEnabled = timeZoneConversion.enabled
```

Comment:
There is no particular reason, maybe I aligned with converter declaration .

I will fix it.